src/Model/Filter/FilterCollection.php

```diff
@@ -2,13 +2,14 @@
 namespace Search\Model\Filter;
 
 use ArrayIterator;
-use Search\Model\Filter\FilterLocatorInterface;
 
 /**
  * FilterCollection
  */
 class FilterCollection implements FilterCollectionInterface
 {
+    use FilterMethodsTrait;
```

Comment:
Why does the filter collection need those?

Reply:
Because with my lazy loading filters support added in previous PR, one directly setups filters in collection class itself inside it's `initialize()` instead of using search manager.